app/controllers/signup_controller.rb

```diff
@@ -15,7 +15,7 @@ class SignupController < ApplicationController
   before_filter :restart_if_missing_signup_state, only: [:verify_email, :password, :social]
 
   # TODO spec this
-  before_filter :exit_signup_if_logged_in, only: [:start, :verify_email, :password, :social, :verify_by_token]
+  before_filter :exit_signup_if_logged_in, only: [:start, :social, :verify_by_token]
```

Comment:
I think we still need to exit signup if logged in for `:verify_email` and `:password`.  Because of the `# TODO spec this` comment nothing may have broken in the specs but I think we still need it.

Reply:
Looks to me like we do have a spec for the exit_signup case 
https://github.com/openstax/accounts/blob/master/spec/features/user_signs_up_spec.rb#L363-L371

I've removed TODOs